test/types/string/psahabu/perf/allocate.chpl

```diff
@@ -0,0 +1,45 @@
+use Time;
+
+config const n = 10000000;
+config const timing = true;
+
+// concatenate
+var tConcat: Timer;
+if timing then tConcat.start();
+var concat: string;
+for i in 1..n do
+  concat = "concat is short for " + "concatenation";
```

Comment:
I think it would be good to also concatenate with 'i', just to make each iteration unique.